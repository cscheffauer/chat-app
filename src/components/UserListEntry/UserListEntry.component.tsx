import React from 'react';

import './UserListEntry.scss';

interface Props {
	key: String;
	username: String;
}

const UserListEntry: React.FunctionComponent<Props> = ({ username }) => {
	return (
		<div className='userlistentry'>
			<p className='username'>{username}</p>
		</div>
	);
};

export default UserListEntry;
