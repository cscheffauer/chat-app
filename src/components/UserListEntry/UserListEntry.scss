@import '../../variables';

.userlistentry {
	width: 100%;
	height: 8vh;
	display: table;
	border-bottom: $border-style-tabs;
	p {
		display: table-cell;
		vertical-align: middle;
		padding-left: 5vw;
	}
}
