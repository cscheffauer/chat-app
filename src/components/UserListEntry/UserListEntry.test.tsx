import React from 'react';
import { shallow } from 'enzyme';
import UserListEntry from './UserListEntry.component';

it('expect to render UserListEntry component', () => {
    const userListEntryWrapper = shallow(<UserListEntry key={'1'} username={'Max Mustermann'} />);
    expect(userListEntryWrapper).toMatchSnapshot();
})
