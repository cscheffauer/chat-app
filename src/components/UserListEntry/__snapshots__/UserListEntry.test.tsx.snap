// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render UserListEntry component 1`] = `
<div
  className="userlistentry"
>
  <p
    className="username"
  >
    Max Mustermann
  </p>
</div>
`;
